src/index.ts

```diff
@@ -103,6 +103,11 @@ export type ConcurrentlyOptions = Omit<BaseConcurrentlyOptions, 'abortSignal' |
      * If not defined, no argument replacing will happen.
      */
     additionalArguments?: string[];
+
+    /**
+     * This command should be run multiple times, for each of the provided matrices.
```

Comment:
Can you please update this comment? It looks like it's assuming some context from elsewhere.